main.go

```diff
@@ -26,12 +24,7 @@ func main() {
 	}
 
 	if debugMode {
-		err := plugin.Debug(context.Background(), "vmware/tanzu/tmc", opts)
```

Comment:
This is changed due to the new linter version?

Reply:
No, this is part of Terraform SDK update. `Debug` function will soon be deprecated.